.github/workflows/release.yml

```diff
@@ -14,3 +14,4 @@ jobs:
     with:
       package-name: aws.secretmanager
       package-org: ballerinax
+      additional-release-flags: "-x test"
```

Comment:
Don't we need to run the tests before releasing?

Reply:
Fixed with [8d17eb468f23e4606f63e5b5e11d6c048e711688](https://github.com/ballerina-platform/module-ballerinax-aws.secretmanager/pull/11/commits/8d17eb468f23e4606f63e5b5e11d6c048e711688)